scripts/miniapps/HPGMG.md

```diff
@@ -33,7 +32,8 @@ and set the recommended environment variables, e.g. for MPICH, etc.
    % make -j3 -C build
```

Comment:
All of the other READMEs, you added to the "Build" line, "(in the directory xxx)".
You should add it here to make them consistent.